library/core/src/result.rs

```diff
@@ -2101,14 +2101,76 @@ impl<T, E, F: ~const From<E>> const ops::FromResidual<Result<convert::Infallible
 {
     #[inline]
     #[track_caller]
-    fn from_residual(residual: Result<convert::Infallible, E>) -> Self {
+    default fn from_residual(residual: Result<convert::Infallible, E>) -> Self {
         match residual {
             Err(e) => Err(From::from(e)),
         }
     }
 }
 
+// FIXME(bgr360): how to properly add this change that depends on compiler
+// functionality that's not yet in the beta?
+#[cfg(not(bootstrap))]
```

Comment:
I think you did it right. The release team has a whole process for how they remove cfg(bootstrap) directives

https://forge.rust-lang.org/release/process.html?highlight=bootstrap#master-bootstrap-update-t-2-day-tuesday

Reply:
ok dope